docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1797,6 +1797,8 @@ Refer to https://www.jaegertracing.io/docs/1.16/sampling/#client-sampling-config
 
 Can be set with the environment variable `JAEGER_SAMPLER_TYPE`.
 
+_Setting `sampler_type` in the `tracing.opentelemetry` section will override this setting._
```

Comment:
```suggestion
_To override this setting, enter `sampler_type` in the `tracing.opentelemetry` section._
```

Reply:
@hairyhenderson , wanted to check with you about the above suggestion before I replicate that wording convention to the docs below.